test-suite/shared/src/test/scala/org/scalajs/testsuite/scalalib/SymbolTest.scala

```diff
@@ -23,60 +23,64 @@ class SymbolTest {
     def expectEqual(sym1: Symbol, sym2: Symbol): Unit = {
       assertTrue(sym1 eq sym2)
       assertEquals(sym2, sym1)
-      assertEquals(sym2, sym1)
       assertEquals(sym2.##, sym1.##)
     }
 
-    expectEqual('ScalaJS, Symbol("ScalaJS"))
-    expectEqual('$, Symbol("$"))
-    expectEqual('-, Symbol("-"))
+    expectEqual(Symbol("ScalaJS"), Symbol("ScalaJS"))
+    expectEqual(Symbol("$"), Symbol("$"))
+    expectEqual(Symbol("-"), Symbol("-"))
+
+    val scalajs = Symbol("ScalaJS")
+    expectEqual(scalajs, Symbol("ScalaJS"))
 
     val `42` = Symbol("42")
-    val map = Map[Symbol, Any](Symbol("ScalaJS") -> "Scala.js", '$ -> 1.2, `42` -> 42)
-    assertEquals("Scala.js", map('ScalaJS))
+    val map = Map[Symbol, Any](Symbol("ScalaJS") -> "Scala.js", Symbol("$") -> 1.2, `42` -> 42)
+    assertEquals("Scala.js", map(Symbol("ScalaJS")))
     assertEquals(1.2, map(Symbol("$")))
     assertEquals(42, map(Symbol("42")))
     assertEquals(42, map(`42`))
   }
 
   @Test def should_support_name(): Unit = {
-    val scalajs = 'ScalaJS
+    val scalajs = Symbol("ScalaJS")
 
     assertEquals("ScalaJS", scalajs.name)
     assertEquals("$", Symbol("$").name)
-    assertEquals("$$", '$$.name)
-    assertEquals("-", '-.name)
-    assertEquals("*", '*.name)
+    assertEquals("$$", Symbol("$$").name)
+    assertEquals("-", Symbol("-").name)
+    assertEquals("*", Symbol("*").name)
     assertEquals("'", Symbol("'").name)
     assertEquals("\"", Symbol("\"").name)
   }
 
   @Test def should_support_toString(): Unit = {
-    val scalajs = 'ScalaJS
+    val scalajs = Symbol("ScalaJS")
 
     val toStringUsesQuoteSyntax = {
       scalaVersion.startsWith("2.10.") ||
       scalaVersion.startsWith("2.11.") ||
       scalaVersion.startsWith("2.12.") ||
       scalaVersion == "2.13.0" ||
       scalaVersion == "2.13.1" ||
-      scalaVersion == "2.13.2"
+      scalaVersion == "2.13.2" ||
+      scalaVersion.startsWith("0.") || // Dotty version start with "0." at the moment
+      scalaVersion.startsWith("3.") // Future scala 3
```

Comment:
This will become invalid as soon as Dotty upgrades its standard library to 2.13.3. Therefore I think it's better to put this PR on hold until that happens, so we don't introduce more need to change stuff here.

Reply:
Ok. I saw now that it will change on the next version of standard library.
I'm go remove those to two lines.